dlt-connector/src/client/IotaClient.ts

```diff
@@ -0,0 +1,51 @@
+import { ClientBuilder } from '@iota/client'
+import { MessageWrapper } from '@iota/client/lib/types'
+
+import { CONFIG } from '@/config'
+
+const client = new ClientBuilder().node(CONFIG.IOTA_API_URL).build()
+
+/**
+ * send data message onto iota tangle
+ * use CONFIG.IOTA_COMMUNITY_ALIAS for index
+ * @param {string} message - the message as utf based string, will be converted to hex automatically from @iota/client
+ * @return {Promise<MessageWrapper>} the iota message typed
+ */
+function sendDataMessage(message: string): Promise<MessageWrapper> {
```

Comment:
Nur ein Vorschlag:
wenn du die Methode sendDataMessage() zum Senden nennst, wäre dann für das Empfangen der Methodenname getDataMessage() nicht konsistenter?
Oder 
wenn es um das Senden und Empfangen im IotaClient geht, dann wäre send/receiveIotaMessage() aussagekräftig oder in Kurzform einfach nur send/receiveMessage()

Reply:
Das schon, aber es wäre technisch inkorrekt. Während die send Funktion nur Daten, bzw. Indiced Messages versenden kann, kann man mit der receive methode alle Arten von Iota Messages empfangen, also auch die normalen Iota Transaktionen. 
Ich habe es geändert auf sendMessage und receiveMessage.